molgenis-das/src/main/java/org/molgenis/das/impl/RepositoryRangeHandlingDataSource.java

```diff
@@ -181,7 +172,7 @@ else if (patients.containsKey(valuePatient))
 					++score;
 				}
 
-				feature = createDasFeature(valueStart, valueStop, valueIdentifier, valueName, valueDescription,
+				feature = createDasFeature(valueStart, valueStop, entity.getIdValue().toString(), entity.getLabelValue(), valueDescription,
```

Comment:
we used to use "GenomicDataSettings.Meta.ATTRS_IDENTIFIER" as the id, instead of taking the entity's idValue. In that situation it could be null, and that is indeed not the case for the idAttribute of an entity.